src/frontend/hooks/use-record/params-to-form-data.ts

```diff
@@ -37,6 +37,11 @@ function paramsToFormData(params: Record<string, any>): FormData {
       }
       return formData.set(key, FORM_VALUE_EMPTY_OBJECT)
     }
+    // Convert Date fields to UTC timezone
+    if (Date.parse(value)) {
+      const date = new Date(value)
+      return formData.set(key, date.toUTCString())
```

Comment:
`toISOString` would probably be better since it's a standard format

Reply:
Yes, I agree